docs/docs/api-ref.md

```diff
@@ -257,5 +257,45 @@ Source files: server/endpoint/users_endpoint.py, models/user_item.py
 
 ## Groups
 
-Source files: server/endpoint/groups_endpoint.py, models/group_item.py,
+Get Groups
 
+```
+groups = server.groups.get()
+```
+Returns two objects - the groups and the pagination item. To access info about groups
+
+```
+group_list = groups[0]
+```
+
+The group object contains the same information as the standard XML from REST API. For Sites with more than one group, you can access each individual group like this.
+
+```
+#To see the attributes
+dir(group_list[i])
+
+#use these attributes (domain_name, id, name, users)
```

Comment:
If this isn't doc'd yet (which I'm sure it isn't?) can you list out the group class and it's attributes with a brief one-liner.

OR, if you'd like, submit a docstring for those classes first, and then update the docs?